    void \u141d(int, int) throws java.io.IOException
    {
        java.lang.Throwable $r2;
        byte[] $r1;
        java.lang.NullPointerException r3;
        util.h.xy.cy.mf r0;
        byte $b2;
        int $i0, $i1, $i3;
        boolean $z0, $z1;

        r0 := @this: util.h.xy.cy.mf;

        $i0 := @parameter0: int;

        $i1 := @parameter1: int;

        $z0 = 0;

        if $i1 >= 31 goto label01;

        $i0 = $i0 | $i1;

        virtualinvoke r0.<util.h.xy.cy.mf: void \u02ce(int)>($i0);

        goto label06;

     label01:
        $i0 = $i0 | 31;

        virtualinvoke r0.<util.h.xy.cy.mf: void \u02ce(int)>($i0);

        if $i1 >= 128 goto label02;

        $z1 = 1;

        goto label03;

     label02:
        $z1 = 0;

     label03:
        if $z1 == 1 goto label05;

        $r1 = newarray (byte)[5];

        $i0 = $i1 & 127;

        $b2 = (byte) $i0;

        $i0 = 4;

        $r1[4] = $b2;

        $i3 = <util.h.xy.cy.mf: int \u02bb>;

        $i3 = $i3 + 13;

        $i3 = $i3 % 128;

        <util.h.xy.cy.mf: int \u02bc> = $i3;

     label04:
        $i1 = $i1 >> 7;

        $i0 = $i0 - 1;

        $i3 = $i1 & 127;

        $i3 = $i3 | 128;

        $b2 = (byte) $i3;

        $r1[$i0] = $b2;

        if $i1 > 127 goto label04;

        $i1 = 5 - $i0;

        virtualinvoke r0.<util.h.xy.cy.mf: void \u02cf(byte[],int,int)>($r1, $i0, $i1);

        goto label06;

     label05:
        $i0 = <util.h.xy.cy.mf: int \u02bb>;

        $i0 = $i0 + 53;

        $i0 = $i0 % 128;

        <util.h.xy.cy.mf: int \u02bc> = $i0;

        virtualinvoke r0.<util.h.xy.cy.mf: void \u02ce(int)>($i1);

     label06:
        $i0 = <util.h.xy.cy.mf: int \u02bb>;

        $i0 = $i0 + 99;

        $i1 = $i0 % 128;

        <util.h.xy.cy.mf: int \u02bc> = $i1;

        $i0 = $i0 % 2;

        if $i0 == 0 goto label07;

        $z0 = 1;

     label07:
        if $z0 == 1 goto label10;

     label08:
        r3 = new java.lang.NullPointerException;

        specialinvoke r3.<java.lang.NullPointerException: void <init>(java.lang.String)>("This statement would have triggered an Exception: specialinvoke $u7#28.<java.lang.Object: int hashCode()>()");

        throw r3;

     label09:
        $r2 := @caughtexception;

        throw $r2;

     label10:
        return;

        catch java.lang.Throwable from label08 to label09 with label09;
    }
